browser/brave_content_browser_client.cc

```diff
@@ -318,6 +320,19 @@ void BindBraveSearchDefaultHost(
   }
 }
 
+void MaybeBindSkusSdkImpl(
+    content::RenderFrameHost* const frame_host,
+    mojo::PendingReceiver<skus::mojom::SkusSdk> receiver) {
+  auto* context = frame_host->GetBrowserContext();
+  auto* profile = Profile::FromBrowserContext(context);
+  // Skus functionality not supported in private / Tor / guest windows
+  if (brave::IsRegularProfile(profile)) {
```

Comment:
you should not have logic about which profiles to use here, that belongs  in the service factory

Reply:
Fixed in https://github.com/brave/brave-core/pull/10847/commits/780676bbebb2283387763229a4592c4eb4eadb9d